docs/api-guide/pagination.md

```diff
@@ -84,24 +85,45 @@ Note that the `paginate_queryset` method may set state on the pagination instanc
 
 Let's modify the built-in `PageNumberPagination` style, so that instead of include the pagination links in the body of the response, we'll instead include a `Link` header, in a [similar style to the GitHub API][github-link-pagination].
 
+```py
+    from rest_framework.pagination import PageNumberPagination
+    from rest_framework.response import Response
+    from rest_framework.utils.urls import remove_query_param, replace_query_param
+
+
     class LinkHeaderPagination(pagination.PageNumberPagination):
         def get_paginated_response(self, data):
-            next_url = self.get_next_link()
            previous_url = self.get_previous_link()
+            link = '<{}>; rel="{}"'
 
-            if next_url is not None and previous_url is not None:
-                link = '<{next_url}; rel="next">, <{previous_url}; rel="prev">'
-            elif next_url is not None:
-                link = '<{next_url}; rel="next">'
-            elif previous_url is not None:
-                link = '<{previous_url}; rel="prev">'
-            else:
-                link = ''
+            first = self.get_first_link()
+            prev = self.get_previous_link()
+            next = self.get_next_link()
+            last = self.get_last_link()
```

Comment:
Not wildly fussed either way. Broadly I think it's a little more code than we need for the example.